app/routes.py

```diff
@@ -1,42 +1,56 @@
 from io import BytesIO
 import os.path
-from flask import Response, make_response, jsonify, abort
+from flask import Response
+from flask import make_response
+from flask import jsonify
+from flask import abort
+from flask import current_app as capp
 from PIL import Image
+
 from app import app
 from app.helpers.check_functions import check_color_channels
 from app.helpers.route import prefix_route
 from app.helpers import make_error_msg
 
 # add route prefix
-app.route = prefix_route(app.route, '/v4/color/')
+app.route = prefix_route(app.route, '/v<int:ver>/color')
 
 
 @app.route('/checker')
-def checker_page():
+def checker_page(ver):
     """
-    Just a route for the health check
-    :return: OK with a 200 status code
+    Just a route for the health check.
+    :param ver: integer value of the service's version (inherited from prefix_route).
+    :return: OK with a 200 status code or raise error in case of unsupported version.
     """
+    if ver > 4:
```

Comment:
I'd make the `4` a variable, we'll probably reset the counting for the internal services to 1

Reply:
Good point. I added "check_version()" to "check_functions.py" and also defined MAX_ALLOWED_VERSION = 4 in "check_functions.py". In "routes.py" all that is left is "check_version(ver)".